src_rebuild/GAME/C/XMPLAY.H

```diff
@@ -90,7 +90,7 @@ typedef struct _XMCHANNEL {
 	u_char dat;
 	u_char datold;
 	u_char eff;
-	u_char not;
+	u_char not_;
```

Comment:
I believe `not` is a keyword in C++11, Visual Studio seems to handle this some other way.